### PR TITLE
DROOLS-2373 : BRL 'is contained in the (comma separated) list' does not produce multiselect dropdown

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2373

https://github.com/kiegroup/drools/pull/2189
https://github.com/kiegroup/kie-wb-common/pull/2359
https://github.com/kiegroup/drools-wb/pull/1025

### CODE REVIEW

File: kie-wb-common-widgets/kie-wb-decorated-grid-widget/src/main/java/org/kie/workbench/common/widgets/decoratedgrid/client/widget/cells/AbstractProxyPopupDropDownListBox.java [hunk 1]
```diff
@@ -50,10 +47,16 @@ protected void selectItem(final ListBox listBox) {
...
+    private final ListBox listBox;
+    private final AbstractProxyPopupDropDownEditCell proxy;
+    private String[][] items;
 
-    public AbstractProxyPopupDropDownListBox(final AbstractProxyPopupDropDownEditCell proxy) {
+    public AbstractProxyPopupDropDownListBox(final AbstractProxyPopupDropDownEditCell proxy,
+                                             final String operator) {
 
         this.listBox = new ListBox();
+        this.listBox.setMultipleSelect(OperatorsOracle.operatorRequiresList(operator));
```
Comment (Rikkola) [inline kie-wb-common-widgets/kie-wb-decorated-grid-widget/src/main/java/org/kie/workbench/common/widgets/decoratedgrid/client/widget/cells/AbstractProxyPopupDropDownListBox.java]: The big change is this line. In other locations I just pass around the operator.

**Comment by Rikkola** [comment]
Looks like I still need to add an unit test for the change.

**Comment by Rikkola** [comment]
Ok. Test added.

**Comment by Rikkola** [comment]
@jomarko Ok these changes add support for the group select for rule templates. That editor didn't have the multiselect option for lists before.

**Comment by Rikkola** [comment]
It has been a while. I doubt there are conflicts, but best to play it safe.

Jenkins please retest this.

**Comment by jomarko** [comment]
@Rikkola @manstis hi, this seems to me as fine to be merged. Is there anything that I am missing and block us from merging? Asking because we have to create 7.14.x cherry pick until Friday.

**Pull request merged**